plugins/xkcd.py

```diff
@@ -45,5 +43,5 @@ def xkcd_url(match):
 
 @hook.command()
 def xkcd(text):
-    """<search term> - Search for xkcd comic matching <search term>"""
+    """xkcd <search term> - Search for xkcd comic matching <search term>"""
```

Comment:
Also your docstring should not contain the command itself, that is added dynamically

Reply:
I haven't touched this since October, when I made the minor URL change. I'm not sure what happened here, but I haven't committed anything in here in a while.